src/gui/resources/GazeboDrawer.qml

```diff
@@ -165,6 +172,34 @@ Rectangle {
     }
   }
 
+  /**
+   * About dialog
+   */
+  Dialog {
+    id: aboutDialog
+    title: "Ignition Gazebo"
+
+    modal: true
+    focus: true
+    parent: ApplicationWindow.overlay
+    width: parent.width / 4
+    x: (parent.width - width) / 2
+    y: (parent.height - height) / 2
+    closePolicy: Popup.CloseOnEscape
+    standardButtons: StandardButton.Ok
+
+    Text {
+      anchors.fill: parent
+      id: aboutMessage
+      wrapMode: Text.Wrap
+      verticalAlignment: Text.AlignVCenter
+      color: Material.theme == Material.Light ? "black" : "white"
+      textFormat: Text.RichText
+      text: AboutDialogHandler.getVersionInformation()
+      onLinkActivated: AboutDialogHandler.openURL(link)
```

Comment:
I haven't investigated why, but the links are not clickable for me. Are they working for you?

Reply:
I believe that it is an issue related to QtQuick. The links aren't styled as expected and weren't opening in the default browser. I have made a workaround to this issue by adding the `linkActivated` signal.

Reply:
Got it! I realized that the link wasn't working for me because I was inside a Docker container :upside_down_face: I tried this now on the host machine and the links work correctly! Thanks!